packages/astro/src/astro-components/interactive/InteractiveUI.astro

```diff
@@ -0,0 +1,42 @@
+---
+interface Props {
+  [key: string]: unknown
+  component: 'sign-in' | 'sign-up' | 'organization-list' | 'organization-profile' | 'organization-switcher' | 'user-button' | 'user-profile'
+}
+
+import { customAlphabet, urlAlphabet } from "nanoid";
+
+const safeId = customAlphabet(urlAlphabet, 10)();
+
+const props = {
+  ...Astro.props,
+}
```

Comment:
Do we need this ? I think we are accessing `Astro.props` below directly

Reply:
Still using this in new update, but destructured since the `component` prop is not a clerk UI element option